2018-04-13 00:33:47,874 [INFO] 1523597627: -----------------------------Experiment ID 1523597627 begins--------------------------
2018-04-13 00:33:47,874 [INFO] 1523597627: Configuration of the experiment: no_experiments: 1
2018-04-13 00:33:47,874 [INFO] 1523597627: Configurations of the experiment: n_max_iters_experiment: 100
2018-04-13 00:33:47,874 [INFO] 1523597627: Configurations of the experiment: n_agents: 4
2018-04-13 00:33:47,874 [INFO] 1523597627: Configurations of the experiment: n_agents_tracking: 1
2018-04-13 00:33:47,874 [INFO] 1523597627: XXXXXXX                                                                        XXX  
X       X    X XXXXX  XXXXXX XXXXX     X   X    X XXXXXX X    X  XXXXX        X   X 
X        X  X  X    X X      X    X    X   XX  XX X      XX   X    X         X     X
XXXXX     XX   X    X XXXXX  X    X    X   X XX X XXXXX  X X  X    X         X     X
X         XX   XXXXX  X      XXXXX     X   X    X X      X  X X    X         X     X
X        X  X  X      X      X   X     X   X    X X      X   XX    X          X   X 
XXXXXXX X    X X      XXXXXX X    X    X   X    X XXXXXX X    X    X           XXX  


2018-04-13 00:33:47,877 [INFO] src.utils.generate_init: Initializing old configuration for grid and agents
2018-04-13 00:33:47,879 [INFO] src.utils.generate_init: Finished creating arena and agents. One item consumed due to overlap. Current item count is 28
2018-04-13 00:33:47,879 [INFO] src.estimation.ABU_estimator_noapproximation: Calling parent ABU via inheritance. This is ABU_noapproximation
2018-04-13 00:33:47,887 [INFO] src.estimation.ABU_estimator: ABU Estimator class initialized with 10 agents and working on estimating parameter view_radius with a resolution of 10
2018-04-13 00:33:47,887 [INFO] src.estimation.ABU_estimator: ABU polynomial info: resolution: 10, llpoly_degree: 9, priorpoly_degree: 9, postpoly_degree: 9, refitdensity: 20
2018-04-13 00:33:47,887 [INFO] 1523597627: iter 0 in experiment 0
2018-04-13 00:33:47,910 [ERROR] root: Experiment failed
Traceback (most recent call last):
  File "/home/manish/Awesomestuff/AdHocTeamWork/AdHoc2/experiments/experiment_master_nocp_nomcts.py", line 163, in <module>
    estimated_type = np.argmax(abu.model_evidence[-1])
IndexError: list index out of range
2018-04-13 00:33:47,940 [INFO] twilio.http_client: POST Request: https://api.twilio.com/2010-04-01/Accounts/AC03c6c74162a11f07ffb0170da406f3c2/Messages.json
2018-04-13 00:33:47,940 [INFO] twilio.http_client: PAYLOAD: {'Body': "Experiment with args Namespace(settingsfolder='/home/manish/Awesomestuff/AdHocTeamWork/AdHoc2/data/e2') exception list index out of range ! Check logs!", 'To': '+15125022558', 'From': '+15126438645'}
2018-04-13 00:33:48,411 [INFO] twilio.http_client: POST Response: 201 {"sid": "SM4e2b9e4e95db40a0801ec9ea14c4cf48", "date_created": "Fri, 13 Apr 2018 05:33:48 +0000", "date_updated": "Fri, 13 Apr 2018 05:33:48 +0000", "date_sent": null, "account_sid": "AC03c6c74162a11f07ffb0170da406f3c2", "to": "+15125022558", "from": "+15126438645", "messaging_service_sid": null, "body": "Sent from your Twilio trial account - Experiment with args Namespace(settingsfolder='/home/manish/Awesomestuff/AdHocTeamWork/AdHoc2/data/e2') exception list index out of range ! Check logs!", "status": "queued", "num_segments": "2", "num_media": "0", "direction": "outbound-api", "api_version": "2010-04-01", "price": null, "price_unit": "USD", "error_code": null, "error_message": null, "uri": "/2010-04-01/Accounts/AC03c6c74162a11f07ffb0170da406f3c2/Messages/SM4e2b9e4e95db40a0801ec9ea14c4cf48.json", "subresource_uris": {"media": "/2010-04-01/Accounts/AC03c6c74162a11f07ffb0170da406f3c2/Messages/SM4e2b9e4e95db40a0801ec9ea14c4cf48/Media.json"}}
